advocacy_docs/supported-open-source/patroni/02-installation.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Installation'
+description: 'Instructions for installing Patroni on supported systems'
+tags:
+  - installation
+  - Patroni
+---
+
+### Available platforms
+
+Patroni packages are provided in the PGDG **apt** and **yum** repositories.
+
+See the [Platform Compatibility](https://www.enterprisedb.com/resources/platform-compatibility#epas) reference for the supported OS list (only **Linux x86-64 (amd64)** currently).
+
+#### Debian/Ubuntu
+
+To install Patroni, configure the [PostgreSQL](https://www.postgresql.org/download/linux/ubuntu/) **apt** repository and then run:
+
+```bash
+$ sudo apt-get install -y patroni
+```
+
+See the [Debian10 Quick Start guide](04-debian10_quick_start/#Patroni) for a more detailed configuration example.
+
+#### RHEL/CentOS
+
+Patroni can be installed from the [PostgreSQL](https://yum.postgresql.org/) **yum** repository, and requires Extra Packages for Enterprise Linux ([EPEL](https://fedoraproject.org/wiki/EPEL)).
+
+Once you configured the repositories, run the following command to install Patroni and its dependencies for etcd:
+
+```bash
+$ sudo dnf install -y epel-release
+$ sudo dnf install -y patroni patroni-etcd
+```
+
+See the [RHEL8 Quick Start guide](03-rhel8_quick_start/#Patroni) for a more detailed configuration example.
+
+#### Python package installer
```

Comment:
Should we mention installing Patroni through pip?

@kelpoole @tomjohnkinedb ?

Reply:
As long as we only support etcd, apt and yum packages should be fine (I didn't verify all the platforms mentioned in the epas platform compatibility list though).
Pip is the only way to install Patroni when relying on the internal raft consensus rather than etcd currently.